linux-armv7a/crosstool-ng.config

```diff
@@ -538,26 +565,30 @@ CT_CC_GCC_SHOW=y
 CT_CC_GCC_PKG_KSYM="GCC"
 CT_GCC_DIR_NAME="gcc"
 CT_GCC_USE_GNU=y
+# CT_GCC_USE_ORACLE is not set
 CT_GCC_USE="GCC"
 CT_GCC_PKG_NAME="gcc"
 CT_GCC_SRC_RELEASE=y
 # CT_GCC_SRC_DEVEL is not set
 CT_GCC_PATCH_ORDER="global"
-# CT_GCC_V_12 is not set
-CT_GCC_V_11=y
+# CT_GCC_V_13 is not set
+CT_GCC_V_12=y
+# CT_GCC_V_11 is not set
 # CT_GCC_V_10 is not set
 # CT_GCC_V_9 is not set
 # CT_GCC_V_8 is not set
 # CT_GCC_V_7 is not set
 # CT_GCC_V_6 is not set
-CT_GCC_VERSION="11.3.0"
+CT_GCC_VERSION="12.3.0"
 CT_GCC_MIRRORS="$(CT_Mirrors GNU gcc/gcc-${CT_GCC_VERSION}) $(CT_Mirrors sourceware gcc/releases/gcc-${CT_GCC_VERSION})"
 CT_GCC_ARCHIVE_FILENAME="@{pkg_name}-@{version}"
 CT_GCC_ARCHIVE_DIRNAME="@{pkg_name}-@{version}"
 CT_GCC_ARCHIVE_FORMATS=".tar.xz .tar.gz"
 CT_GCC_SIGNATURE_FORMAT=""
-CT_GCC_12_or_older=y
-CT_GCC_older_than_12=y
+CT_GCC_13_or_older=y
+CT_GCC_older_than_13=y
+CT_GCC_later_than_12=y
```

Comment:
This seems to be in conflict with `CT_GCC_VERSION="12.3.0"` and `CT_GCC_V_12=y`.

But this was similarly out of sync before your change:
```
CT_GCC_VERSION="11.3.0"
CT_GCC_VERSION="11.3.0"
```
and
```
CT_GCC_12_or_older=y
CT_GCC_older_than_12=y
```

I may be misunderstanding something, but it just stuck out as odd to me.

Reply:
But 12.3.0 is *older* than gcc 13, so the config CT_GCC_13_*or_older* should be yes as it is. This config was automatically generated by crosstools, though